src/content/resources/server-endpoints-file-streaming.json

```diff
@@ -0,0 +1,6 @@
+{
+    "tags": ["API Endpoints"],
+    "title": "Streaming a file in Astro",
+    "link": "https://astropatterns.dev/p/gems/server-endpoints-file-streaming",
+    "updated": "unknown"
```

Comment:
Hey Florian!  This linked resource (astropatterns.dev/p...) doesn't have a date listed.  
I thought i saw something in other resources with this... but let me check now

Reply:
I reached out to the author on x.com.

Reply:
If the author doesn't answer, I would just try to estimate the date. If we go with what [Wayback Machine](https://web.archive.org/web/collections/20240000000000*/https://astropatterns.dev/p/gems/server-endpoints-file-streaming) knows, it would be September 2024 :)

Reply:
He didn't reply.  I figure just estimate.  and September 1, 2024 seems good.  I'll make that change now

Reply:
Oh - what about the tag?  I have it as "API Endpoint" ... that's not great.  What about API?  which would be a new one.  Or, use the existing "DB"

Reply:
I'm going to reach out too, the category is fine

Reply:
Thank you!